app/views/upload.scala.html

```diff
@@ -8,6 +8,19 @@
         @progressIndicator(Messages("upload.progress"))
 
         <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
+        <div id="uploadError" class="govuk-error-summary hide" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
```

Comment:
Minor point, but we're using kebab-case for CSS class names (partly because we're using [BEM](http://getbem.com/introduction/) as a naming scheme), so HTML IDs should be in kebab-case too. There might be elements in camelCase elsewhere in the codebase, but we should be as consistent as possible.

Reply:
Changed. Sorry I forgot we had decided on using BEM